Makefile

```diff
@@ -44,6 +44,7 @@ build: $(cmds)
 
 $(images):
 	docker build -f build/docker/$@.Dockerfile $(DOCKER_ARGS) -t $@:$(TAG) .
+	docker tag $@:$(TAG) $@:latest
```

Comment:
generally, `:latest` is something that is supposed to be used with "latest stable". Those images are in theory more development images, so, maybe `:devel` ?